zap/src/main/java/org/zaproxy/zap/extension/api/CoreAPI.java

```diff
@@ -204,6 +213,7 @@ private enum ScanReportType {
     private static final String PARAM_INDEX = "index";
     private static final String PARAM_FILENAME = "fileName";
     private static final String PARAM_CONTENTS = "fileContents";
+    private static final String PARAM_BASE_PACKAGE = "basepackage";
```

Comment:
`baseName`

Reply:
Better rename the constant and other places (endpoint description should use just "name", it's not necessarily a pacakge).